modules/dnn/src/layers/fast_convolution/fast_convolution.cpp

```diff
@@ -61,7 +70,7 @@ Ptr<FastConv2d> initFastConv2d(
         int ksize = Hk*Wk;
 
         // this code aims to let memory fit with vector size.
-        int padded_ksize = ((ksize + FAST_VEC_NLANES-1) / FAST_VEC_NLANES) * FAST_VEC_NLANES;
+        int padded_ksize = ((ksize + VEC_NLANES-1) / VEC_NLANES) * VEC_NLANES;
```

Comment:
> `((ksize + VEC_NLANES-1) / VEC_NLANES) * VEC_NLANES`

FYI, `alignSize(ksize, VEC_NLANES)` for 2**n or `roundUp(ksize, VEC_NLANES)` for others